src/wiki/getting-started/install-of-alternatives.md

```diff
@@ -39,6 +39,11 @@ If you don't want to search, and install these mods manually, then try the [Fabu
 
 See the [MultiMC/MultiMC (auto-update)](https://fabulously-optimized.gitbook.io/modpack/readme/install-instructions#multimc) guide for help installing it. These instructions will work for Prism Launcher too.
 
+### <img src="https://cdn.modrinth.com/data/BYfVnHa7/9ebb06896c378743a875e58f60baaa8a8d5938e9.png" height="20"> Simply Optimized
+
+[Simply Optimized](https://modrinth.com/modpack/sop) is a modpack designed with just ptimization in mind. SO has better out of the box performance than Fabulously Optimized, but it doesn't come with the QoL mods or full OptiFine parity you would see in Fabulously Optimized, so you're expected to add any additional mods you want yourself.
```

Comment:
```suggestion
[Simply Optimized](https://modrinth.com/modpack/sop) is a modpack designed to just hace optimization mods. SO might better out of the box performance than Fabulously Optimized because of the lack of QoL features, but it doesn't come with any QoL mods or any OptiFine parity mod you would see in Fabulously Optimized, so you'll need to install them manually when needed
```